website/docs/docs/build/sample-flag.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "About the sample flag"
+description: "Use the sample flag to lower development time and reduce warehouse spend."
+sidebar_label: "The sample flag"
+pagination_next: null
+pagination_prev: "docs/build/empty-flag"
+---
+
+# About the `--sample` flag
+
+Large data sets can drastically increase build times and reduce how quickly dbt developers can build and test new code. The dbt `--sample` flag can help to reduce build times when developing and warehouse spend by running dbt in sample mode. Sample mode enables you to address cases where you don't need to build the entire model during the development or CI cycle but include enough data to validate the outputs. 
+
+Sample mode builds upon the [`--empty` flag](/docs/build/empty-flag) by taking the validation of semantic results a step further by including a sampling of data from the model(s) in your development schema. It won't solve every scenario; for example, there are cases where not all joins will be populated. However, it presents a viable solution for faster building, testing, and validating many strategies. 
+
+The `--sample` flag will become more robust over time, but it only supports time-based sampling for now.
+
+## Using the `--sample` flag
+
+The `--sample` flag is available for the [`run`](/reference/commands/run) and [`build`](/reference/commands/build) commands, and when used, sample mode generates filtered refs and sources. Since it's using time-based sampling, if you have refs like {{ ref('some_model') }} being sampled, you need to set [`event_time`](/reference/resource-configs/event-time) for `some_model` to the field that will be used as the timestamp. 
```

Comment:
```suggestion
The `--sample` flag is available for the [`run`](/reference/commands/run) and [`build`](/reference/commands/build) commands, and when used, sample mode generates filtered refs and sources. Since it's using time-based sampling, if you have refs like `{{ ref('some_model') }}` being sampled, you need to set [`event_time`](/reference/resource-configs/event-time) for `some_model` to the field that will be used as the timestamp. 
```